src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] It would be nice if you import your Calculator component here to follow the project requirements because App.js is the main file to handle all the components 👍